python-threatexchange/threatexchange/signal_type/pdq.py

```diff
@@ -50,6 +50,8 @@ def match_file(self, file: pathlib.Path) -> t.List[signal_base.SignalMatch]:
         return self.match_hash(pdq_hash)
 
     def match_hash(self, signal_str: str) -> t.List[signal_base.SignalMatch]:
+        if len(signal_str) != BITS_IN_PDQ / 4:
```

Comment:
As I was thinking through why someone would provide a hash with the number of hex characters for the signal_str, I was thinking this might be a copy and paste error. Is this a case where we would want to print an error message to the user to let them know we are ignoring the hash because it's not the right length?

Reply:
This was added because without it, if you give it an photo md5 or pdq_ocr string using the -H flag, for 'match photo' it will assert because it tries to match against all types photo hashes. I can add a comment.